### PR TITLE
Add tasks 206-208

### PR DESCRIPTION
**Pull request by zaetsa12**

### CODE REVIEW

**Review by Nataly997 — APPROVED** [review]

**Review by FeloniukM — APPROVED** [review]

File: Projects/Team1/AnnaZaiets/Task2/Task2_2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            string str;
+            int a, b, c;
+            Console.WriteLine("Enter your number from 0 to 100:");
+            str = Console.ReadLine();
+            a = Convert.ToInt32(str);
+
+            if (a < 10)
+            {
+                b = 1;
+                Console.WriteLine("Чисел" + b);
```
Comment (romanukhaav) [inline Projects/Team1/AnnaZaiets/Task2/Task2_2/Program.cs:23]: Use only English plese

**Review by romanukhaav — APPROVED** [review]

**Review by dazeforlife — APPROVED** [review]

File: Projects/Team1/AnnaZaiets/Task2/Task2/Program.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+using System.Threading.Tasks;
+
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+
+            //Console.WriteLine($"The sum of numbers: {firstNum + secondNum + thirdNum}\nThe product of numbers: {firstNum * secondNum * thirdNum}");
```
Comment (romanukhaav) [inline Projects/Team1/AnnaZaiets/Task2/Task2/Program.cs:14]: Do we need this comment?

File: Projects/Team1/AnnaZaiets/Task2/Task2/Program.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            //1. Напишіть програму, яка перевіряє число, введене з клавіатури на парність.
+
+            string str;
+            int a;
+            Console.WriteLine("Enter your number:");
+            str = Console.ReadLine();
+            a = Convert.ToInt32(str);
+
```
Comment (romanukhaav) [inline Projects/Team1/AnnaZaiets/Task2/Task2/Program.cs:22]: Try to avoid redundant empty rows

**Review by romanukhaav — CHANGES_REQUESTED** [review]
please, fix all comments

**Review by Sasha380 — APPROVED** [review]

File: Projects/Team1/AnnaZaiets/Task2/Task2/Program.cs (hunk 2 above)
Comment (zaetsa12) [reply Projects/Team1/AnnaZaiets/Task2/Task2/Program.cs:14]: no, I forgot to delete

File: Projects/Team1/AnnaZaiets/Task2/Task2_2/Program.cs (hunk 1 above)
Comment (zaetsa12) [reply Projects/Team1/AnnaZaiets/Task2/Task2_2/Program.cs:23]: oops , ок

**Review by SvitlanaLN — COMMENTED** [review]
image
Here you may use label or recursion to return to the beginning of the program and repeat again:

else
            {
               // Console.WriteLine("Введіть число від 0 до 100");
                Main(args);
            }

**Comment by zaetsa12** [comment]
> image Here you may use label or recursion to return to the beginning of the program and repeat again:
> 
> else { // Console.WriteLine("Введіть число від 0 до 100"); Main(args); }
> 
> or
> 
> else { // Console.WriteLine("Введіть число від 0 до 100"); goto label1; }

thanks for the interesting thought

**Review by SvitlanaLN — APPROVED** [review]

**Review by romanukhaav — APPROVED** [review]

**Pull request merged**